core/src/main/java/org/apache/calcite/sql/SqlUtil.java

```diff
@@ -261,6 +261,32 @@ public static boolean isLiteralChain(SqlNode node) {
     }
   }
 
+  /**
+   * Unparses a call to an operator which has special syntax.
+   *
+   * @param operator The operator
+   * @param writer   Writer
+   * @param call     List of 0 or more operands
+   */
+  public static void unparseSpecialSyntax(SqlOperator operator,
+      SqlWriter writer,
+      SqlCall call) {
+    final List<SqlNode> operands = call.getOperandList();
+    switch (operator.getKind()) {
+    case INSERT:
+      assert operands.size() == 3 || operands.size() == 4;
+      final SqlInsert insert = new SqlInsert(call.getParserPosition(),
+          (SqlNodeList) operands.get(0), operands.get(1), operands.get(2),
+          operands.size() == 3 ? null : (SqlNodeList) call.getOperandList().get(3));
```

Comment:
why `null` and why not reusing `operands` in this line ?

Reply:
what's the cases for `operands.size() == 3` and `operands.size() == 4`

Reply:
The size of operands is always 4, right?

Reply:
I mean: 
1. why `null`, is there a better value to pass
2. call.getOperandList() => operands

Reply:
@yanlin-Lynn @zinking 
Thanks.The size of operands is always 4, update the code.